manifests/p/ppy/osu/2024.1224.1/ppy.osu.locale.zh-CN.yaml

```diff
@@ -0,0 +1,112 @@
+  Platform
+    Enable beatmap exports on iOS (ppy/osu#31037 by @frenzibyte)
+    Bump OpenTabletDriver to 0.6.5.0 (ppy/osu-framework#6465 by @jamesbt365)
+    Pulls in detection & support improvements for tens of tablet models.Smaller changes
+    - Fix IPC not working in release (ppy/osu#31000 by @LittleEndu)
+    - Prefer HDTV / Rec. 709 colorspace for videos with HD resolution and unspecified colorspace (ppy/osu-framework#6403 by @frenzibyte)
+    - Fix unobserved exceptions on named pipe shutdown (ppy/osu-framework#6447 by @peppy)
+    - Add descriptions to osu! file associations in iOS (ppy/osu#31035 by @frenzibyte)
+    - Add support for presenting/opening files on iOS (ppy/osu-framework#6448 by @frenzibyte)
+    - Use temporary storage for file exports on iOS (ppy/osu#31039 by @frenzibyte)
+    - Fix multiple instances of a game being allowed to start concurrently (ppy/osu-framework#6452 by @peppy)
+    - Bring back iOS application delegates and fix many issues (ppy/osu-framework#6453 by @frenzibyte)
+    - Fix iOS audio session allowing other background audio to mix with game audio (ppy/osu-framework#6457 by @frenzibyte)
+    - Fix disabling hold-to-right-click not cancelling in-progress actions (ppy/osu-framework#6459 by @frenzibyte)
+    - Affects mobile devices. Noticed in osu! gameplay, where if you press resume from the pause overlay, a right click is always triggered at the point where the button was clicked:
+    - Fix iOS app configuration missing certain specifications (ppy/osu#31162 by @frenzibyte)
+    - Fixes some iOS submission guideline failures.
+  Results
+    Allow using "quick retry" shortcut in more cases from results screen (ppy/osu#31108 by @peppy)
+    Fixes "quick retry" hotkey not working for autoplay, but also allows using it for other replays when the replay is already guaranteed to be local (ie. on the second playback).Skin Editor
+    Fix player settings overlay appearing while in skin editor (ppy/osu#31105 by @peppy)
+    Also stops the settings appearing if the mouse is too far away from the window.Skinning
+    - Fix lookups of hit circle slider pieces potentially using wrong source skin (ppy/osu#31052 by @peppy)
+    - Visible on some maps with partial beatmap skins included:
+    Add recommended difficulty numerical value near filter in beatmap listing (ppy/osu#31101 by @Joehuu)
+    - Fix top score statistics section total score display being terminally broken (ppy/osu#31041 by @bdach)
+    - Fixes total score displaying completely wrong and mangled values on languages that use spaces as thousands separators.
+    - Expose high precision mouse toggle when searching for "sensitivity" and other keywords (ppy/osu#31139 by @peppy)
+    - Add missing mania tooltip overlay for 4k and 7k (ppy/osu#31084 by @NicholasChin28)
+  New Contributors
+    - @NicholasChin28 made their first contribution in #31084
+  Full Changelog: https://github.com/ppy/osu/compare/2024.1208.0...2024.1219.2"

```

Comment:
```suggestion
```suggestion
ReleaseNotes: |
  Difficulty Calculation
    ✓ Refactor TaikoDifficultyCalculator and add DifficultStrain attributes (osu#31191)作者: Lawtrohux
    ✓ Increase accscalingshift and include countok in hit proportion (osu#31195)作者: Lawtrohux
    ✓ Remove Conditional SkillRating nerf to Converts (osu#31196)作者: Lawtrohux
    ✓ Simplify osu! high-bpm acute angle jumps bonus (osu#30902)作者: stanriders
    + Implement Reading Skill into osu!taiko (osu#31208)作者: Lawtrohux
    ✓ Change slider drop penalty to use actual number of difficult sliders, fix slider drop penalty being too lenient (osu#31055)作者: stanriders
    ✓ Use lastAngle when nerfing repeated angles on acute bonus (osu#31245)作者: tsunyoku
  Editor
    ✓ Fix being able to click through slider tail drag handles (osu#31184)作者: peppy
    ✓ Change beat snap divisor adjust defaults to be Ctrl+Scroll instead of Ctrl+Shift+Scroll (osu#31147)作者: peppy
  Framework
    ✓ Fix BindableList methods crashing with non-IList collections (osu-framework#6466)作者: smoogipoo
    ✓ Move "global" scroll-adjusts-volume to a per-screen component-based implementation (osu#31146)作者: peppy
    + Implement getting OpenGL iOS main framebuffer on SDL3 (osu-framework#6468)作者: Susko3
    ✓ Fix typo in main menu tip (osu#31224)作者: frenzi作者:te
    ✓ Use Accelerate framework for texture image processing on iOS and fix premultiplication issues (osu-framework#6470)作者: frenzi作者:te
    ✓ Shave allocation overheads in recent iOS texture loading code (osu-framework#6471)作者: frenzi作者:te
  Gameplay
    ✓ Skip the pause cooldown when in intro / break time (osu#31143)作者: peppy
    + Implement skinnable mod display (osu#30993)作者: Tom94
    Gameplay (osu!)
      ✓ Fix some hitcircle sprites missing in certain skins since last release (osu#31254)作者: peppy
    Gameplay (osu!mania)
      ✓ Fix "spinner" conversion for mania-specific beatmaps (osu#30984)作者: smoogipoo
  Main Menu
    ✓ Load seasonal backgrounds without requiring being logged in (osu#31198)作者: peppy
      No reason not to.
    + Add christmas / seasonal mode (osu#31206)作者: peppy
    ✓ Slightly reduce background brightness at main menu when seasonal lighting is active (osu#31258)作者: peppy
    ✓ Fix adjusting volume using scroll wheel not working during intro (osu#31259)作者: peppy
  Platform
    ✓ Update game structure with the returned iOS application delegates (osu#31152)作者: frenzi作者:te
    ✓ Fix logo animation not rendering correctly on iOS (osu-resources#351)作者: frenzi作者:te
    ✓ Fix hardware cursor showing on iPadOS when it shouldn't (osu#31226)作者: frenzi作者:te
  Settings
    ✓ Save "featured artists" filter to user settings (osu#31238)作者: frenzi作者:te
      Also sets it to always on for iOS in preparation for a larger scale release.
    + Add ability to rename the current skin (osu#31220)作者: normalid-awa
      In settings for now, very temporary.
  Song Select
    + Implement ability to mark beatmap as played (osu#31138)作者: bdach
      Would you believe it that this button that has been there for literal years never did anything? Also available via context menu.
  Tournament
    ✓ Hide !mp commands from tournament streaming chat (osu#31150)作者: peppy
  Full Changelog: https://github.com/ppy/osu/compare/2024.1219.2...2024.1224.1
ReleaseNotesUrl: https://osu.ppy.sh/home/changelog/lazer/2024.1224.1
```
```